frontend/src/routes/Send.tsx

```diff
@@ -129,10 +127,6 @@ function Send() {
       </header>
```

Comment:
Seems like it would help if the <Close /> component would accept some onClose callback that overrides the default behavior of navigating to root, that way we can pass cleanup logic when necessary